src/shared/lib/GenericAssayUtils.ts

```diff
@@ -0,0 +1,113 @@
+import client from 'shared/api/cbioportalClientInstance';
```

Comment:
lets prepare for future when there are multiple assays the use generic assay.  should be folder genericAssayUtils and each type should have its own file

Reply:
Yes, that's a great idea! Will do!